packages/plugins/rubygem-smart_proxy_discovery_image/rubygem-smart_proxy_discovery_image.spec

```
@@ -1,64 +1,89 @@
+# template: smart_proxy_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%{!?_root_datadir:%global _root_datadir %{_datadir}}
+%{!?_root_localstatedir:%global _root_localstatedir %{_localstatedir}}
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
 %global gem_name smart_proxy_discovery_image
+%global plugin_name discovery_image
 
-%global foreman_proxy_dir %{_datarootdir}/foreman-proxy
+%global foreman_proxy_min_version 1.24
+%global foreman_proxy_dir %{_root_datadir}/foreman-proxy
+%global foreman_proxy_statedir %{_root_localstatedir}/foreman-proxy
 %global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
-%global foreman_proxy_settingsd_dir %{_sysconfdir}/foreman-proxy/settings.d
+%global foreman_proxy_settingsd_dir %{_root_sysconfdir}/foreman-proxy/settings.d
+%global smart_proxy_dynflow_bundlerd_dir %{!?scl:/opt/theforeman/tfm/root}%{_datadir}/smart_proxy_dynflow_core/bundler.d
 
-Name: rubygem-%{gem_name}
+Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 1.0.9
-Release: 1%{?dist}
-Summary: Discovery image (node) plugin for Foreman's smart proxy
+Release: 3%{?foremandist}%{?dist}
+Summary: FDI API for Foreman Smart-Proxy
 Group: Applications/Internet
-License: GPLv3
-URL: https://github.com/theforeman/smart_proxy_discovery_image
+License: GPL-3.0
+URL: http://github.com/theforeman/smart_proxy_discovery_image
```

In git master this has been fixed (https://github.com/theforeman/smart_proxy_discovery_image/commit/82ca40744e6985c666a0af7db47c039dc617f72e)
```suggestion
URL: https://github.com/theforeman/smart_proxy_discovery_image
```